war/package.json

```diff
@@ -15,9 +15,12 @@
     "build": "yarn prod",
     "start": "yarn dev --watch",
     "test": "jest --ci --config=jest.config.js",
-    "lint:js": "eslint . --ext js -f checkstyle -o target/eslint-warnings.xml",
-    "lint:css": "stylelint src/main/less --custom-formatter node_modules/stylelint-checkstyle-reporter/index.js -o target/stylelint-warnings.xml",
+    "lint:js": "eslint . --ext js",
+    "lint:js-ci": "eslint . --ext js -f checkstyle -o target/eslint-warnings.xml",
+    "lint:css": "stylelint src/main/less",
+    "lint:css-ci": "stylelint src/main/less --custom-formatter stylelint-formatter.js -o target/stylelint-warnings.xml",
```

Comment:
stylelint custom-formatters are loaded from an absolute path which doesn't work with PnP, I did a workaround of creating a file at a known path which then loads the library correctly through module resolution:

https://github.com/stylelint/stylelint/issues/6100